src/Tickets/Commerce/Gateways/PayPal/Hooks.php

```diff
@@ -167,6 +168,17 @@ public function handle_action_refresh_user_info() {
 		$saved = $merchant->save_user_info( $user_info );
 	}
 
+	/**
+	 * Handles the refreshing of the webhook on PayPal for this site/merchant.
+	 *
+	 * @todo  Display some message when refreshing user info.
+	 * @since TBD
+	 *
+	 */
+	public function handle_action_refresh_webhook() {
+		$updated = $this->container->make( Webhooks::class )->create_or_update_existing();
```

Comment:
Do we want to return this?

Reply:
No in this case it's an action URL so no need to return.